### ISSUE TITLE
Cannot use import statement outside a module

### ISSUE THREAD

**Issue by hqwlkj** [ant-design/ant-design-icons#619]
![image](https://github.com/ant-design/ant-design-icons/assets/12181423/1f4d8686-ddd0-47c8-8ea9-64d9b8b614fa)

Next.js

```
Operating System:
  Platform: darwin
  Arch: x64
  Version: Darwin Kernel Version 20.6.0: Thu Jul  6 22:12:47 PDT 2023; root:xnu-7195.141.49.702.12~1/RELEASE_X86_64
Binaries:
  Node: 18.18.2
  npm: 9.8.1
  Yarn: 1.22.19
  pnpm: 7.16.0
Relevant Packages:
  next: 14.1.0
  eslint-config-next: 14.1.0
  react: 18.2.0
  react-dom: 18.2.0
  typescript: 5.2.2
Next.js Config:
  output: standalone

```

antd 
```

"@ant-design/icons": "^5.2.6",
"antd": "^5.13.1",


```

**Comment by hqwlkj** [comment]
> ![image](https://private-user-images.githubusercontent.com/12181423/300308360-1f4d8686-ddd0-47c8-8ea9-64d9b8b614fa.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcwMTk2MTksIm5iZiI6MTcwNzAxOTMxOSwicGF0aCI6Ii8xMjE4MTQyMy8zMDAzMDgzNjAtMWY0ZDg2ODYtZGRkMC00N2M4LThlYTktNjRkOWI4YjYxNGZhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA0VDA0MDE1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBmMzQ4NTQyODc1MmI0Y2Y0NDdjZGIxMmE0NTM2MjUwZDJlYTZkYjdiZWQ2YTRjMDVjYWIzNmNkZWM2NTlkY2ImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.Mf-KYabuZp3y0sXsWJpvBo20rLtcSp0JCnWXDcXfu6k)
> 
> Next.js
> 
> ```
> Operating System:
>   Platform: darwin
>   Arch: x64
>   Version: Darwin Kernel Version 20.6.0: Thu Jul  6 22:12:47 PDT 2023; root:xnu-7195.141.49.702.12~1/RELEASE_X86_64
> Binaries:
>   Node: 18.18.2
>   npm: 9.8.1
>   Yarn: 1.22.19
>   pnpm: 7.16.0
> Relevant Packages:
>   next: 14.1.0
>   eslint-config-next: 14.1.0
>   react: 18.2.0
>   react-dom: 18.2.0
>   typescript: 5.2.2
> Next.js Config:
>   output: standalone
> ```
> 
> antd
> 
> ```
> 
> "@ant-design/icons": "^5.2.6",
> "antd": "^5.13.1",
> ```

@afc163  可以帮忙看看什么原因吗？

**Comment by Pasov** [comment]
I have same issue.
Is this issue still unresolved?

antd 

```
"@ant-design/icons": "^5.3.0",
"antd": "^5.14.1",
```

**Comment by kkk360** [comment]
If you are using nextjs 14, You can modify the  next.config.js file, add 
`const nextConfig = { 
 ... , 
 transpilePackages: ["@ant-design/icons"],
}`

**Comment by Pasov** [comment]
@kkk360 Thanks. It works :) Is that in the official document?

**Comment by afc163** [comment]
Fixed in `@ant-design/icons@5.4.0`: https://github.com/ant-design/ant-design-icons/pull/653

**Comment by morganney** [comment]
But then you removed the fix in https://github.com/ant-design/ant-design-icons/pull/655.
